CMakeLists.txt

```diff
@@ -250,16 +248,31 @@ if(WIN32)
 
     ${phd_src_dir}/cameras/ArtemisHSCAPI.h
     ${phd_src_dir}/cameras/ArtemisHSCAPI.cpp
+
+    ${phd_src_dir}/cam_qhy.cpp
+    ${phd_src_dir}/cam_qhy.h
   )
 elseif(UNIX AND (NOT APPLE))
   set(cam_SRC
       ${cam_SRC}
       ${phd_src_dir}/cam_qhy5.h
       ${phd_src_dir}/cam_qhy5.cpp
+
+      ${phd_src_dir}/cam_qhy.cpp
+      ${phd_src_dir}/cam_qhy.h
      )
+elseif(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
```

Comment:
I was not sure how to do this in a nice way. Suggestions are welcome if you have something better than this.

Reply:
Actually, you can revert his change and update thirdparty.cmake as follows:
Find the if block that says `if(IS_DIRECTORY ${PHD_PROJECT_ROOT_DIR}/cameras/qhyccdlibs/linux/${qhyarch})`
and update it so that the body of that if block is excluded for FreeBSD.
By eliminating `add_definitions(-DHAVE_QHY_CAMERA=1)` the qhy code is excluded from the build (without needing to futz around with the list of camera source files here.)